dbt-cta/stackadapt/models/0_ctes/account_native_ads_stats_ab3.sql

```diff
@@ -46,7 +46,7 @@ select
         'vcomp_75',
         'vcomp_95',
         'click_url',
-        array_to_string('creatives'),
+        '(SELECT STRING_AGG(TO_JSON_STRING(STRUCT(JSON_EXTRACT_SCALAR(element, \'$.url\') as url, JSON_EXTRACT_SCALAR(element, \'$.size\') as size)), \',\' ORDER BY JSON_EXTRACT_SCALAR(element, \'$.url\'), JSON_EXTRACT_SCALAR(element, \'$.size\')) FROM UNNEST(creatives) as element)',
```

Comment:
i had to make this change because the new data coming in actually changed the hash order, and led to an issue where the hash in the migration was no longer matching the hashes historically. this fixed that!